service-stubs/identity/org.wso2.carbon.identity.governance.stub/src/main/resources/IdentityGovernanceAdminService.wsdl

```diff
@@ -195,4 +224,4 @@
             <http:address location="https://localhost:9443/services/IdentityGovernanceAdminService.IdentityGovernanceAdminServiceHttpsEndpoint/"/>
         </wsdl:port>
     </wsdl:service>
-</wsdl:definitions>
+</wsdl:definitions>
```

Comment:
add new line EOF

Reply:
Fixed in commit 68014963860d7ab4d47f35e9e9901b95df6d62b4